### PR TITLE
First Rails Application: Reworded section 3.3, adding note about heroku for WLS

### PR DESCRIPTION
**Pull request by IrritatedEllipses**
As per @I3uckwheat I've reworded a bit of section 3.3 in my first rails application as well as adding a note in 4.9 on manually opening the rails app if `heroku open` fails to open the browser.

### CODE REVIEW

**Comment by mindovermiles262** [comment]
Hey @IrritatedEllipses it looks like you need to update your repo.  Make sure you have `upstream` added, then `git pull upstream master` and then recommit to your local branch. This should fix the Merge Conflict. PM me with any Questions

**Comment by IrritatedEllipses** [comment]
Thanks @mindovermiles262 ! Looks like I had a wrong upstream selected. Got it fixed and recommited.

File: web_development_101/installfest_first_rails_app.md [hunk 1]
```diff
@@ -487,21 +487,15 @@ git commit -m "initial commit"
...
+First we'll open [GitHub.com](https://github.com/) in the browser and sign in (if you aren't already). Next, look for your profile picture in the upper right hand corner, click the + symbol next to it, and create a `new repository`. Give the repository a name (Maybe `my_first_rails_app`?), then click `Create Repository` below the form. For your first project you shouldn't need to mess with any of the other options, but they're good to read over for future projects!
 
-GitHub should have taken you to a page that has some commands listed.
+On your `Create Repository` page you'll see various commands listed. We know that we want to use SSH to connect to GitHub so make sure `SSH` has been selected. Look for the section that says "... or push an existing repository from the command line" and click the icon to the right of it to copy the text to your clipboard. Then paste it into the terminal we have open (Using CTRL + SHIFT + V in Linux or CTRL + V in WSL).
```
Comment (I3uckwheat) [inline web_development_101/installfest_first_rails_app.md]: Bold: **make sure `SSH` has been selected** maybe?

I can see people skimming this too quickly

**Comment by IrritatedEllipses** [comment]
I declined to add the blink tag @I3uckwheat  so fond of and, instead, bolded the entire line

**Pull request merged**